pandamium_datapack/data/pandamium/functions/misc/teleport/teleport.mcfunction

```diff
@@ -0,0 +1,18 @@
+execute if score <tp_y> variable matches ..-1 run scoreboard players set <tp_y> variable 0
+execute if score <tp_y> variable matches 257.. run scoreboard players set <tp_y> variable 256
```

Comment:
Should I keep this?

Reply:
no, I don't think it's necessary, if for some reason the tp function gets called with those parameters that's a problem with the caller, not the tp function